programs/clearing_house/src/controller/orders.rs

```diff
@@ -1248,13 +1271,60 @@ pub fn fulfill_order_with_match(
         return Ok((0_u128, false));
     }
 
+    let amm_wants_to_make = match taker.orders[taker_order_index].direction {
+        PositionDirection::Long => market.amm.net_base_asset_amount < 0,
+        PositionDirection::Short => market.amm.net_base_asset_amount > 0,
+    };
+
+    let base_asset_amount_left_to_fill = if market.amm.amm_jit && amm_wants_to_make {
+        println!("amm jiting");
```

Comment:
we could probably just derive this from the fill records? also if you want this to be in solana logs, you need to use msg!